### ISSUE TITLE
Influence functions

### ISSUE THREAD

**Issue by jklaise** [SeldonIO/alibi#16]
This method requires taking Hessians of the loss function, so is not model-agnostic. Implementations will rely on ML frameworks with autograd. We can do Tensorflow for 0.1.0 and Pytorch afterwards.
